x/concentrated-liquidity/internal/math/tick.go

```diff
@@ -180,9 +185,9 @@ func powTenBigDec(exponent sdk.Int) osmomath.BigDec {
 	return osmomath.OneDec().Quo(osmomath.NewBigDec(10).Power(osmomath.NewBigDec(exponent.Abs().Int64())))
 }
 
-// GetMinAndMaxTicksFromExponentAtPriceOne determines min and max ticks allowed for a given exponentAtPriceOne value
+// getMinAndMaxTicksFromExponentAtPriceOne determines min and max ticks allowed for a given exponentAtPriceOne value
```

Comment:
Isnt it safe to expose these get functions publically?

Reply:
I need help with a better way to implement this but the reason for this is as follows:

Other modules need to utilize this function, and having this in internal prevents this even if it is public. So what I did to circumnavigate this is make it private in internal, create a clone of it in the home of the cl module but make that public and utilize that public function in other modules. 

Any recommendation for a cleaner implementation would be appreciated.

Reply:
> Other modules need to utilize this function, and having this in internal prevents this even if it is public.

Are we getting circular reference for this?

Reply:
Yes, normally this wouldn't be an issue but because it has to be used in the internal package, it has to be defined both within internal as well as outside. There has to be another way to do this that I am just not aware of though

Reply:
Your idea from the call worked, thanks :)

https://github.com/osmosis-labs/osmosis/pull/4051/commits/297ca1c4d2bac22d0edaccfa1e92ea151cf8c18b

Reply:
For reviewer context, the suggestion on the call was to keep the internal function private and have a getter method exporting it to public at the CL package layer

Reply:
Sorry, should have added the context, thanks!